src/typings/sharify.d.ts

```diff
@@ -35,7 +37,7 @@ declare module "sharify" {
       readonly ENABLE_YOUR_ALERTS_PAGE: boolean
       readonly FACEBOOK_APP_NAMESPACE: string
       readonly FACEBOOK_ID: string
-      readonly FEATURE_FLAGS: Record<string, boolean>
+      readonly FEATURE_FLAGS: FeatureFlags
```

Comment:
@mc-jones - It looks like this is the culprit -- env vars can only be core types (string, bool, number, etc) and its not liking that record with complex types embedded in, leading to cascading failures. 

If we can get away with `Record<string, boolean>` and live with a bit of duplication that should be fine.

Reply:
Interestingly, the issue ultimately was the import statement, which I never would have guessed given the error message - so thanks for narrowing it down! This seems related to how TS handles [namespaces](https://stackoverflow.com/questions/39040108/import-class-in-definition-file-d-ts); moving the import within the namespace declaration works but doesn't seem like a great solution. I don't love having this typed as an `any` but this seems consistent with how we handle some of the other more complex data and allows us to move this forward. I will look into improving this in subsequent iterations.

One option would be similar to what @dimatretyak mentioned [here](https://github.com/artsy/force/pull/9545#discussion_r819496567) and move the interface to a shared typing file.

Reply:
> One option would be similar to what @dimatretyak mentioned https://github.com/artsy/force/pull/9545#discussion_r819496567 and move the interface to a shared typing file.

It's probably a bit overkill IMO, since once we have the data and it's in the `SystemContext`, we have the types. Reading things at the sharify level outside of the react hook will likely be pretty rare (I _think_).